_includes/pbc/all/install-features/202404.0/install-dynamic-multistore.md

```diff
@@ -0,0 +1,1399 @@
+
+{% endinfo_block %}
+
+This document describes how to install [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html).
+
+## Install feature core
+
+Follow the steps below to install the Dynamic Multistore feature core.
+### Prerequisites
+
+Follow the steps below to install the Dynamic Store feature:
+
+### 1) Make sure that Country, Locale and Currency modules migrated to the required major version
+
+In order to make Dynamic Store work, new major versions of 3 modules must be updated.
+
+{% info_block warningBox %}
+
+The minimum PHP version required to get or update the following modules is PHP 8.1 or higher.
+
+{% endinfo_block %}
+
+| NAME     | VERSION |
+|----------|---------|
+| Country  | ^4.0.0  |
+| Locale   | ^4.0.0  |
+| Currency | ^4.0.0  |
+
+Migration guides can be found here:
+[Country](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/upgrade-modules/upgrade-the-country-module.html)
+[Locale](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/upgrade-modules/upgrade-the-locale-module.html)
+[Currency](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/upgrade-modules/upgrade-the-currency-module.html)
+
+### 2) Install the required modules using Composer
+
+To continue feature integration, overview and install or update the required feature:
+
+| NAME | VERSION |
+| --- | --- |
+|Spryker Core  | {{page.version}} |
+
+
+Install the required modules using Composer:
+
+```bash
+composer require "spryker-feature/spryker-core":"{{page.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+| --- | --- |
+| CountryDataImport | vendor/spryker/country-data-import |
+| СountryGui | vendor/spryker/country-gui |
+| LocaleDataImport | vendor/spryker/locale-data-import |
+| LocaleGui | vendor/spryker/locale-gui |   
+| StoreContextGui | vendor/spryker/store-context-gui |
+| StoreDataImport | vendor/spryker/store-data-import |
+| StoreGui | vendor/spryker/store-gui |
+
+{% endinfo_block %}
+
+
+### 3) Set up configuration
+
+{% info_block warningBox "Configuration stores.php" %}
+
+Before the introduction of dynamic store, configuration settings were maintained in the file `config/Shared/stores.php`. With the activation of dynamic store, these settings are now managed in the database, rendering the files `config/Shared/stores.php` and `config/Shared/default_store.php` obsolete.
+
+The default configuration will be imported using new data import modules, including StoreDataImport, LocaleDataImport, and CountryDataImport. These modules will populate the necessary configuration details in the database.
```

Comment:
Could you also add to "will be imported using new data import modules, including StoreDataImport, LocaleDataImport, and CountryDataImport" smth like "and other feature related imports". What I mean that we e.g. have Currentcy that alsy might have a migration part if I'm not wrong and customers sometime miss those